documentation/install_from_packages.markdown

```diff
@@ -7,103 +7,70 @@ canonical: "/puppetserver/latest/install_from_packages.html"
 [repodocs]: https://puppet.com/docs/puppet/latest/puppet_platform.html
 [passengerguide]: https://puppet.com/docs/puppet/latest/passenger.html
 
-## System Requirements
-
 Puppet Server is configured to use 2 GB of RAM by default. If you'd like to just play around with an installation on a Virtual Machine, this much memory is not necessary. To change the memory allocation, see [Memory Allocation](#memory-allocation).
 
 > If you're also using PuppetDB, check its [requirements](https://puppet.com/docs/puppetdb/latest/index.html#system-requirements).
 
-## Platforms with Packages
-
-Puppet provides official packages that install Puppet Server 5.1 and all of its prerequisites on x86_64 architectures for the following platforms, as part of [Puppet Platform][repodocs].
-
-### Red Hat Enterprise Linux
+## Install Puppet Server from packages
 
--   Enterprise Linux 6
--   Enterprise Linux 7
+1.  [Enable the Puppet package repositories][repodocs], if you haven't already done so.
 
-### Debian
+2.  Install the Puppet Server package by running:
 
--   Debian 8 (Jessie)
+````
+yum install puppetserver
+````
 
-Java 8 runtime packages do not exist in the standard repositories for Debian 8 (Jessie).  To install Puppet Server on Jessie, [configure the `jessie-backports` repository](https://backports.debian.org/Instructions/), which includes openjdk-8:
+or
 
 ```
-echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
-apt-get update
-apt-get -t jessie-backports install "openjdk-8-jdk-headless"
 apt-get install puppetserver
 ```
 
-If you upgraded on Debian from older versions of Puppet Server, or from Java 7 to Java 8, you must also configure your server to use Java 8 by default for Puppet Server 5.x:
-
-```
-update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
-```
-
-### Ubuntu
-
--   Ubuntu 18.04 (Bionic)
--   Ubuntu 16.04 (Xenial)
-
-On Ubuntu 18.04, enable the [universe repository](https://help.ubuntu.com/community/Repositories/Ubuntu), which contains packages necessary for Puppet Server.
+There is no `-` in the package name.
```

Comment:
@justinstoller Is this correct? As there is a `-` in `apt-get`?.

Reply:
yeah, debian's package manager is called "apt" and it's command line tools are "apt-get" and "apt-cache", etc.

Our package used to be called "puppet-server" (which was the package name in Puppet Server 1.x) but for a long time it's just been "puppetserver".

Reply:
Ah okay great, thanks!